pkgs/servers/sql/materialize/default.nix

```
@@ -0,0 +1,90 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, fetchzip
+, rustPlatform
+, bootstrap_cmds
+, DiskArbitration
+, Foundation
+, cmake
+, libiconv
+, openssl
+, perl
+, pkg-config}:
+
+let
+  fetchNpmPackage = {name, version, hash, js_prod_file, js_dev_file, ...} @ args:
+  let
+    package = fetchzip {
+      url = "https://registry.npmjs.org/${name}/-/${baseNameOf name}-${version}.tgz";
+      inherit hash;
+    };
+
+    static = "./src/materialized/src/http/static";
+    cssVendor = "./src/materialized/src/http/static/css/vendor";
+    jsProdVendor = "./src/materialized/src/http/static/js/vendor";
+    jsDevVendor = "./src/materialized/src/http/static-dev/js/vendor";
+
+    files = with args; [
+      { src = js_prod_file; dst = "${jsProdVendor}/${name}.js"; }
+      { src = js_dev_file;  dst = "${jsDevVendor}/${name}.js"; }
+    ] ++ lib.optional (args ? css_file) { src = css_file; dst = "${cssVendor}/${name}.css"; }
+      ++ lib.optional (args ? extra_file) { src = extra_file.src; dst = "${static}/${extra_file.dst}"; };
+  in
+    lib.concatStringsSep "\n" (lib.forEach files ({src, dst}: ''
+      mkdir -p "${dirOf dst}"
+      cp "${package}/${src}" "${dst}"
+    ''));
+
+  npmPackages = import ./npm_deps.nix;
+in
+rustPlatform.buildRustPackage rec {
+  pname = "materialize";
+  version = "0.7.1";
+  rev = "f4bd159fa73d37d44f8ed3f1db13c0c2ff85566f";
+
+  src = fetchFromGitHub {
+    owner = "MaterializeInc";
+    repo = pname;
+    inherit rev;
+    hash = "sha256-8nonB/KRv4qOGvJhh0v3UwlBzAXfzb3afeCm/7/E0AU=";
+  };
+
+  cargoSha256 = "sha256-6bObwtUGTLG8oCNzRnW31gFhb4yBwqDkoijkivPY1z0=";
```

```suggestion
  cargoSha256 = "sha256-5slgICqLZFqxPymgHvq98BtcD70hfJMr36pvAoQKEJ4=";
```

I still get this hash miss match. 

```
materialize> warning: skipping suspicious writable file '/nix/store/cfziwncdlhvazgckgzkwly4w640p4y7x-materialize-0.7.1-vendor.tar.gz'
error: build of '/nix/store/nd45k5pqkg9z8z3gihqs6d4ggrh5j5jm-materialize-0.7.1-vendor.tar.gz.drv' on 'ssh://clara.r' failed: error: hash mismatch in fixed-output derivation '/nix/store/nd45k5pqkg9z8z3gihqs6d4ggrh5j5jm-materialize-0.7.1-vendor.tar.gz.drv':
                specified: sha256-6bObwtUGTLG8oCNzRnW31gFhb4yBwqDkoijkivPY1z0=
                   got:    sha256-5slgICqLZFqxPymgHvq98BtcD70hfJMr36pvAoQKEJ4=
error: builder for '/nix/store/nd45k5pqkg9z8z3gihqs6d4ggrh5j5jm-materialize-0.7.1-vendor.tar.gz.drv' failed with exit code 1;
```

---

I also just tested this commit on Linux on a brand new EC2 instance and the hash was correct

---

that is the full log:

```
copying path '/nix/store/np1bx9747bxnjrjlphrh3wmririy0zwc-source' from 'http://clara:5000'...
unpacking sources
unpacking source archive /nix/store/np1bx9747bxnjrjlphrh3wmririy0zwc-source
source root is source
patching sources
building
    Updating crates.io index
    Updating git repository `https://github.com/fede1024/rust-rdkafka.git`
    Updating git submodule `https://github.com/edenhill/librdkafka.git`
    Updating git repository `https://github.com/rusoto/rusoto.git`
    Updating git submodule `https://github.com/boto/botocore.git`
    Updating git repository `https://github.com/TimelyDataflow/differential-dataflow.git`
    Updating git repository `https://github.com/MaterializeInc/rust-prometheus.git`
    Updating git repository `https://github.com/TimelyDataflow/timely-dataflow`
    Updating git repository `https://github.com/MaterializeInc/serde-protobuf.git`
 Downloading crates ...
  Downloaded ansi_term v0.11.0
  Downloaded strsim v0.8.0
  Downloaded fallible-streaming-iterator v0.1.9
  Downloaded cfg-if v0.1.10
  Downloaded arrayvec v0.5.1
  Downloaded encoding v0.2.33
  Downloaded criterion v0.3.3
  Downloaded adler v0.2.3
  Downloaded fstrings v0.2.3
  Downloaded crc32fast v1.2.1
  Downloaded crc v1.8.1
  Downloaded futures-util v0.3.13
  Downloaded rand_pcg v0.2.1
  Downloaded paste-impl v0.1.18
  Downloaded phf_codegen v0.8.0
  Downloaded phf v0.8.0
  Downloaded pest v2.1.3
  Downloaded phf_generator v0.8.0
  Downloaded parse-zoneinfo v0.3.0
  Downloaded proc-macro-error-attr v1.0.4
  Downloaded proc-macro-crate v0.1.4
  Downloaded predicates-core v1.0.0
  Downloaded pdqselect v0.1.0
  Downloaded percent-encoding v2.1.0
  Downloaded parse_duration v2.1.0
  Downloaded proptest v0.10.1
  Downloaded const_fn v0.4.5
  Downloaded rand_hc v0.3.0
  Downloaded proc-macro2 v1.0.24
  Downloaded radium v0.5.3
  Downloaded quote v1.0.9
  Downloaded protobuf-codegen v2.17.0
  Downloaded psm v0.1.9
  Downloaded rand_xorshift v0.2.0
  Downloaded same-file v1.0.4
  Downloaded rand_core v0.6.2
  Downloaded predicates-tree v1.0.0
  Downloaded rand_chacha v0.2.2
  Downloaded crossbeam-channel v0.5.0
  Downloaded difference v2.0.0
  Downloaded rand_distr v0.4.0
  Downloaded datadriven v0.5.0
  Downloaded rand v0.7.3
  Downloaded rlimit v0.5.4
  Downloaded rustc_version v0.3.0
  Downloaded stable_deref_trait v1.2.0
  Downloaded serde_regex v1.1.0
  Downloaded semver-parser v0.10.1
  Downloaded slab v0.4.2
  Downloaded redox_syscall v0.2.4
  Downloaded snap v1.0.4
  Downloaded remove_dir_all v0.5.1
  Downloaded dirs-sys-next v0.1.2
  Downloaded sha-1 v0.9.4
  Downloaded getopts v0.2.21
  Downloaded siphasher v0.3.3
  Downloaded socket2 v0.3.19
  Downloaded sha2 v0.9.3
  Downloaded sharded-slab v0.1.1
  Downloaded signal-hook-registry v1.2.0
  Downloaded symbolic-common v8.0.0
  Downloaded subtle v2.2.3
  Downloaded structopt-derive v0.4.14
  Downloaded shell-words v1.0.0
  Downloaded cpuid-bool v0.1.2
  Downloaded bytes v1.0.1
  Downloaded serde_json v1.0.64
  Downloaded doc-comment v0.3.1
  Downloaded stringprep v0.1.2
  Downloaded debugid v0.7.2
  Downloaded csv-core v0.1.10
  Downloaded criterion-plot v0.4.3
  Downloaded stacker v0.1.13
  Downloaded enum-iterator-derive v0.6.0
  Downloaded log v0.4.14
  Downloaded derivative v2.2.0
  Downloaded regex-syntax v0.6.23
  Downloaded rusty-fork v0.3.0
  Downloaded static_assertions v1.1.0
  Downloaded core-foundation v0.9.1
  Downloaded foreign-types-shared v0.1.1
  Downloaded futures-channel v0.3.13
  Downloaded cmake v0.1.41
  Downloaded bytefmt v0.1.7
  Downloaded filetime v0.2.14
  Downloaded arc-swap v0.4.8
  Downloaded duct v0.13.3
  Downloaded fs_extra v1.1.0
  Downloaded abomonation_derive v0.5.0
  Downloaded miniz_oxide v0.4.0
  Downloaded cfg-if v1.0.0
  Downloaded normalize-line-endings v0.3.0
  Downloaded flate2 v1.0.20
  Downloaded encoding-index-tradchinese v1.20141219.5
  Downloaded generic-array v0.14.4
  Downloaded failure v0.1.8
  Downloaded console v0.14.0
  Downloaded reqwest v0.11.1
  Downloaded sysinfo v0.16.4
  Downloaded postgres v0.19.0
  Downloaded structopt v0.3.21
  Downloaded serde v1.0.124
  Downloaded tempfile v3.2.0
  Downloaded openssl-probe v0.1.2
  Downloaded num-iter v0.1.40
  Downloaded tar v0.4.33
  Downloaded parking_lot_core v0.8.3
  Downloaded fast-float v0.2.0
  Downloaded oorandom v11.1.0
  Downloaded tap v1.0.0
  Downloaded ntapi v0.3.4
  Downloaded native-tls v0.2.7
  Downloaded getrandom v0.1.14
  Downloaded rand_hc v0.2.0
  Downloaded termcolor v1.1.2
  Downloaded ipnet v2.3.0
  Downloaded tokio-util v0.6.4
  Downloaded protobuf-codegen-pure v2.17.0
  Downloaded tokio-stream v0.1.4
  Downloaded humansize v1.1.0
  Downloaded include_dir v0.6.0
  Downloaded encoding-index-simpchinese v1.20141219.5
  Downloaded regex v1.4.5
  Downloaded bumpalo v3.2.1
  Downloaded futures v0.3.13
  Downloaded futures-task v0.3.13
  Downloaded askama_derive v0.10.5
  Downloaded num_cpus v1.13.0
  Downloaded futures-macro v0.3.13
  Downloaded form_urlencoded v1.0.0
  Downloaded futures-io v0.3.13
  Downloaded scopeguard v1.1.0
  Downloaded funty v1.0.1
  Downloaded cc v1.0.67
  Downloaded unicode-segmentation v1.6.0
  Downloaded cast v0.2.2
  Downloaded symbolic-demangle v8.0.0
  Downloaded addr2line v0.14.1
  Downloaded foreign-types v0.3.2
  Downloaded encode_unicode v0.3.6
  Downloaded encoding-index-japanese v1.20141219.5
  Downloaded failure_derive v0.1.7
  Downloaded crypto-mac v0.10.0
  Downloaded arbitrary v0.4.7
  Downloaded async-compression v0.3.7
  Downloaded ryu v1.0.5
  Downloaded abomonation v0.7.3
  Downloaded decnumber-sys v0.1.5
  Downloaded crossbeam-deque v0.8.0
  Downloaded serde_cbor v0.11.1
  Downloaded rand v0.8.3
  Downloaded smallvec v1.6.1
  Downloaded semver v0.11.0
  Downloaded security-framework v2.0.0
  Downloaded env_logger v0.8.3
  Downloaded syn v1.0.62
  Downloaded anyhow v1.0.38
  Downloaded enum-kinds v0.5.0
  Downloaded rusqlite v0.24.2
  Downloaded redox_users v0.4.0
  Downloaded digest v0.9.0
  Downloaded proc-macro-hack v0.5.19
  Downloaded ansi_term v0.12.1
  Downloaded fallible-iterator v0.2.0
  Downloaded crossbeam-utils v0.8.3
  Downloaded security-framework-sys v2.0.0
  Downloaded ahash v0.4.6
  Downloaded schannel v0.1.18
  Downloaded encoding-index-korean v1.20141219.5
  Downloaded serde-value v0.7.0
  Downloaded tinytemplate v1.1.0
  Downloaded xattr v0.2.2
  Downloaded wasm-bindgen-futures v0.4.19
  Downloaded globset v0.4.6
  Downloaded walkdir v2.3.1
  Downloaded pprof v0.4.2
  Downloaded try-lock v0.2.2
  Downloaded synstructure v0.12.3
  Downloaded want v0.3.0
  Downloaded tracing-core v0.1.17
  Downloaded proc-macro-error v1.0.4
  Downloaded tower-service v0.3.0
  Downloaded thread_local v1.0.1
  Downloaded pin-utils v0.1.0
  Downloaded url v2.2.1
  Downloaded http v0.2.3
  Downloaded pkg-config v0.3.17
  Downloaded unicode-xid v0.2.0
  Downloaded typenum v1.12.0
  Downloaded unicode-width v0.1.8
  Downloaded wyz v0.2.0
  Downloaded ucd-trie v0.1.3
  Downloaded toml v0.5.8
  Downloaded xml-rs v0.8.0
  Downloaded hex-literal v0.3.1
  Downloaded num-bigint v0.2.6
  Downloaded memchr v2.3.3
  Downloaded bitflags v1.2.1
  Downloaded num-integer v0.1.42
  Downloaded jemallocator v0.3.2
  Downloaded num-complex v0.2.4
  Downloaded winreg v0.7.0
  Downloaded postgres-types v0.2.0
  Downloaded textwrap v0.11.0
  Downloaded postgres-protocol v0.6.0
  Downloaded pin-project v1.0.1
  Downloaded num-traits v0.2.14
  Downloaded num-rational v0.2.3
  Downloaded thiserror-impl v1.0.20
  Downloaded wasm-bindgen-macro-support v0.2.69
  Downloaded num_enum_derive v0.5.1
  Downloaded aws-arn v0.1.1
  Downloaded num_enum v0.5.1
  Downloaded bitvec v0.19.4
  Downloaded humantime v2.0.1
  Downloaded sysctl v0.4.0
  Downloaded postgres-openssl v0.5.0
  Downloaded postgres_array v0.11.0
  Downloaded md5 v0.7.0
  Downloaded pin-project-internal v1.0.1
  Downloaded lexical-core v0.7.4
  Downloaded hashlink v0.6.0
  Downloaded http-body v0.4.0
  Downloaded async-trait v0.1.48
  Downloaded lock_api v0.4.2
  Downloaded itoa v0.4.6
  Downloaded base64 v0.13.0
  Downloaded once_cell v1.5.2
  Downloaded matchers v0.0.1
  Downloaded unicode-normalization v0.1.12
  Downloaded matches v0.1.8
  Downloaded hyper-tls v0.5.0
  Downloaded winapi-util v0.1.3
  Downloaded tracing-log v0.1.2
  Downloaded chrono-tz v0.5.3
  Downloaded backtrace v0.3.56
  Downloaded hex v0.4.3
  Downloaded tracing-subscriber v0.2.16
  Downloaded heck v0.3.1
  Downloaded fnv v1.0.7
  Downloaded tokio-postgres v0.7.0
  Downloaded memmap v0.7.0
  Downloaded lazy_static v1.4.0
  Downloaded autocfg v1.0.1
  Downloaded tower-layer v0.3.1
  Downloaded whoami v1.1.0
  Downloaded memoffset v0.6.1
  Downloaded tokio-native-tls v0.3.0
  Downloaded unicase v2.6.0
  Downloaded atty v0.2.14
  Downloaded wasm-bindgen-backend v0.2.69
  Downloaded uuid v0.8.2
  Downloaded wasi v0.10.0+wasi-snapshot-preview1
  Downloaded wasi v0.9.0+wasi-snapshot-preview1
  Downloaded inotify-sys v0.1.4
  Downloaded linked_hash_set v0.1.4
  Downloaded hyper-openssl v0.9.1
  Downloaded vcpkg v0.2.8
  Downloaded mio v0.7.6
  Downloaded indexmap v1.4.0
  Downloaded askama_shared v0.11.1
  Downloaded httpdate v0.3.2
  Downloaded tracing-attributes v0.1.13
  Downloaded wasm-bindgen v0.2.69
  Downloaded hyper v0.14.4
  Downloaded json v0.12.4
  Downloaded tracing-futures v0.2.5
  Downloaded hmac v0.10.1
  Downloaded bit-vec v0.6.3
  Downloaded indicatif v0.15.0
  Downloaded bit-set v0.5.2
  Downloaded rayon v1.5.0
  Downloaded treeline v0.1.0
  Downloaded csv v1.1.6
  Downloaded miow v0.3.6
  Downloaded tracing v0.1.25
  Downloaded tokio-macros v1.1.0
  Downloaded openssl v0.10.32
  Downloaded askama_escape v0.10.1
  Downloaded hashbrown v0.9.1
  Downloaded shared_child v0.3.4
  Downloaded serde_derive v1.0.124
  Downloaded rustc-demangle v0.1.16
  Downloaded futures-executor v0.3.13
  Downloaded either v1.5.2
  Downloaded opaque-debug v0.3.0
  Downloaded num v0.2.0
  Downloaded include_dir_impl v0.6.0
  Downloaded pin-project-lite v0.2.4
  Downloaded number_prefix v0.3.0
  Downloaded phf_shared v0.8.0
  Downloaded ordered-float v2.1.1
  Downloaded js-sys v0.3.46
  Downloaded thiserror v1.0.20
  Downloaded os_info v3.0.1
  Downloaded vec_map v0.8.1
  Downloaded md-5 v0.9.1
  Downloaded itertools v0.9.0
  Downloaded bincode v1.3.1
  Downloaded httparse v1.3.3
  Downloaded web-sys v0.3.36
  Downloaded assert_cmd v1.0.3
  Downloaded getrandom v0.2.1
  Downloaded nom v6.0.1
  Downloaded mime v0.3.13
  Downloaded instant v0.1.6
  Downloaded inotify v0.9.2
  Downloaded jobserver v0.1.21
  Downloaded wasm-bindgen-shared v0.2.69
  Downloaded nix v0.19.1
  Downloaded object v0.23.0
  Downloaded version_check v0.9.2
  Downloaded half v1.6.0
  Downloaded openssl-sys v0.9.61
  Downloaded glob v0.3.0
  Downloaded byteorder v1.4.2
  Downloaded idna v0.2.0
  Downloaded ppv-lite86 v0.2.10
  Downloaded unicode-bidi v0.3.4
  Downloaded terminal_size v0.1.16
  Downloaded os_pipe v0.9.1
  Downloaded jemalloc-ctl v0.3.3
  Downloaded linked-hash-map v0.5.3
  Downloaded libm v0.2.1
  Downloaded block-buffer v0.9.0
  Downloaded regex-automata v0.1.9
  Downloaded encoding-index-singlebyte v1.20141219.5
  Downloaded rand_core v0.5.1
  Downloaded protobuf v2.17.0
  Downloaded float-cmp v0.8.0
  Downloaded core-foundation-sys v0.8.2
  Downloaded compile-time-run v0.2.11
  Downloaded dirs-next v2.0.0
  Downloaded rayon-core v1.9.0
  Downloaded dec v0.3.3
  Downloaded zeroize v1.1.0
  Downloaded hermit-abi v0.1.6
  Downloaded rand_chacha v0.3.0
  Downloaded quick-error v1.2.3
  Downloaded cpp_demangle v0.3.2
  Downloaded build_const v0.2.1
  Downloaded proc-macro-nested v0.1.6
  Downloaded fstrings-proc-macro v0.2.3
  Downloaded enum-iterator v0.6.0
  Downloaded paste v0.1.18
  Downloaded procfs v0.9.1
  Downloaded futures-core v0.3.13
  Downloaded crossbeam-epoch v0.9.1
  Downloaded clap v2.33.3
  Downloaded wait-timeout v0.2.0
  Downloaded chrono v0.4.19
  Downloaded parking_lot v0.11.1
  Downloaded tokio-openssl v0.6.1
  Downloaded wasm-bindgen-macro v0.2.69
  Downloaded bstr v0.2.14
  Downloaded askama v0.10.5
  Downloaded aho-corasick v0.7.15
  Downloaded libc v0.2.89
  Downloaded gimli v0.23.0
  Downloaded serde_urlencoded v0.7.0
  Downloaded tokio v1.3.0
  Downloaded encoding_index_tests v0.1.4
  Downloaded h2 v0.3.0
  Downloaded zstd-sys v1.4.19+zstd.1.4.8
  Downloaded predicates v1.0.7
  Downloaded honggfuzz v0.5.52
  Downloaded futures-sink v0.3.13
  Downloaded libsqlite3-sys v0.20.1
  Downloaded winapi v0.3.8
  Downloaded encoding_rs v0.8.26
  Downloaded plotters v0.2.12
  Downloaded sasl2-sys v0.1.11
  Downloaded libz-sys v1.1.2
  Downloaded jemalloc-sys v0.3.2
  Downloaded winapi-x86_64-pc-windows-gnu v0.4.0
  Downloaded winapi-i686-pc-windows-gnu v0.4.0
  Downloaded openssl-src v111.14.0+1.1.1j
  Downloaded krb5-src v0.2.4+1.18.2
  Downloaded shlex v0.1.1
   Vendoring abomonation v0.7.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/abomonation-0.7.3) to materialize-0.7.1-vendor.tar.gz/abomonation
   Vendoring abomonation_derive v0.5.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/abomonation_derive-0.5.0) to materialize-0.7.1-vendor.tar.gz/abomonation_derive
   Vendoring addr2line v0.14.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/addr2line-0.14.1) to materialize-0.7.1-vendor.tar.gz/addr2line
   Vendoring adler v0.2.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/adler-0.2.3) to materialize-0.7.1-vendor.tar.gz/adler
   Vendoring ahash v0.4.6 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/ahash-0.4.6) to materialize-0.7.1-vendor.tar.gz/ahash
   Vendoring aho-corasick v0.7.15 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/aho-corasick-0.7.15) to materialize-0.7.1-vendor.tar.gz/aho-corasick
   Vendoring ansi_term v0.11.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/ansi_term-0.11.0) to materialize-0.7.1-vendor.tar.gz/ansi_term-0.11.0
   Vendoring ansi_term v0.12.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/ansi_term-0.12.1) to materialize-0.7.1-vendor.tar.gz/ansi_term
   Vendoring anyhow v1.0.38 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/anyhow-1.0.38) to materialize-0.7.1-vendor.tar.gz/anyhow
   Vendoring arbitrary v0.4.7 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/arbitrary-0.4.7) to materialize-0.7.1-vendor.tar.gz/arbitrary
   Vendoring arc-swap v0.4.8 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/arc-swap-0.4.8) to materialize-0.7.1-vendor.tar.gz/arc-swap
   Vendoring arrayvec v0.5.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/arrayvec-0.5.1) to materialize-0.7.1-vendor.tar.gz/arrayvec
   Vendoring askama v0.10.5 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/askama-0.10.5) to materialize-0.7.1-vendor.tar.gz/askama
   Vendoring askama_derive v0.10.5 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/askama_derive-0.10.5) to materialize-0.7.1-vendor.tar.gz/askama_derive
   Vendoring askama_escape v0.10.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/askama_escape-0.10.1) to materialize-0.7.1-vendor.tar.gz/askama_escape
   Vendoring askama_shared v0.11.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/askama_shared-0.11.1) to materialize-0.7.1-vendor.tar.gz/askama_shared
   Vendoring assert_cmd v1.0.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/assert_cmd-1.0.3) to materialize-0.7.1-vendor.tar.gz/assert_cmd
   Vendoring async-compression v0.3.7 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/async-compression-0.3.7) to materialize-0.7.1-vendor.tar.gz/async-compression
   Vendoring async-trait v0.1.48 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/async-trait-0.1.48) to materialize-0.7.1-vendor.tar.gz/async-trait
   Vendoring atty v0.2.14 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/atty-0.2.14) to materialize-0.7.1-vendor.tar.gz/atty
   Vendoring autocfg v1.0.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/autocfg-1.0.1) to materialize-0.7.1-vendor.tar.gz/autocfg
   Vendoring aws-arn v0.1.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/aws-arn-0.1.1) to materialize-0.7.1-vendor.tar.gz/aws-arn
   Vendoring backtrace v0.3.56 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.56) to materialize-0.7.1-vendor.tar.gz/backtrace
   Vendoring base64 v0.13.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/base64-0.13.0) to materialize-0.7.1-vendor.tar.gz/base64
   Vendoring bincode v1.3.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/bincode-1.3.1) to materialize-0.7.1-vendor.tar.gz/bincode
   Vendoring bit-set v0.5.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/bit-set-0.5.2) to materialize-0.7.1-vendor.tar.gz/bit-set
   Vendoring bit-vec v0.6.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/bit-vec-0.6.3) to materialize-0.7.1-vendor.tar.gz/bit-vec
   Vendoring bitflags v1.2.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/bitflags-1.2.1) to materialize-0.7.1-vendor.tar.gz/bitflags
   Vendoring bitvec v0.19.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/bitvec-0.19.4) to materialize-0.7.1-vendor.tar.gz/bitvec
   Vendoring block-buffer v0.9.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/block-buffer-0.9.0) to materialize-0.7.1-vendor.tar.gz/block-buffer
   Vendoring bstr v0.2.14 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/bstr-0.2.14) to materialize-0.7.1-vendor.tar.gz/bstr
   Vendoring build_const v0.2.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/build_const-0.2.1) to materialize-0.7.1-vendor.tar.gz/build_const
   Vendoring bumpalo v3.2.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/bumpalo-3.2.1) to materialize-0.7.1-vendor.tar.gz/bumpalo
   Vendoring bytefmt v0.1.7 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/bytefmt-0.1.7) to materialize-0.7.1-vendor.tar.gz/bytefmt
   Vendoring byteorder v1.4.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/byteorder-1.4.2) to materialize-0.7.1-vendor.tar.gz/byteorder
   Vendoring bytes v1.0.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/bytes-1.0.1) to materialize-0.7.1-vendor.tar.gz/bytes
   Vendoring cast v0.2.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/cast-0.2.2) to materialize-0.7.1-vendor.tar.gz/cast
   Vendoring cc v1.0.67 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/cc-1.0.67) to materialize-0.7.1-vendor.tar.gz/cc
   Vendoring cfg-if v0.1.10 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/cfg-if-0.1.10) to materialize-0.7.1-vendor.tar.gz/cfg-if-0.1.10
   Vendoring cfg-if v1.0.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/cfg-if-1.0.0) to materialize-0.7.1-vendor.tar.gz/cfg-if
   Vendoring chrono v0.4.19 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/chrono-0.4.19) to materialize-0.7.1-vendor.tar.gz/chrono
   Vendoring chrono-tz v0.5.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/chrono-tz-0.5.3) to materialize-0.7.1-vendor.tar.gz/chrono-tz
   Vendoring clap v2.33.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/clap-2.33.3) to materialize-0.7.1-vendor.tar.gz/clap
   Vendoring cmake v0.1.41 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/cmake-0.1.41) to materialize-0.7.1-vendor.tar.gz/cmake
   Vendoring compile-time-run v0.2.11 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/compile-time-run-0.2.11) to materialize-0.7.1-vendor.tar.gz/compile-time-run
   Vendoring console v0.14.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/console-0.14.0) to materialize-0.7.1-vendor.tar.gz/console
   Vendoring const_fn v0.4.5 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/const_fn-0.4.5) to materialize-0.7.1-vendor.tar.gz/const_fn
   Vendoring core-foundation v0.9.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/core-foundation-0.9.1) to materialize-0.7.1-vendor.tar.gz/core-foundation
   Vendoring core-foundation-sys v0.8.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/core-foundation-sys-0.8.2) to materialize-0.7.1-vendor.tar.gz/core-foundation-sys
   Vendoring cpp_demangle v0.3.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/cpp_demangle-0.3.2) to materialize-0.7.1-vendor.tar.gz/cpp_demangle
   Vendoring cpuid-bool v0.1.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/cpuid-bool-0.1.2) to materialize-0.7.1-vendor.tar.gz/cpuid-bool
   Vendoring crc v1.8.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/crc-1.8.1) to materialize-0.7.1-vendor.tar.gz/crc
   Vendoring crc32fast v1.2.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/crc32fast-1.2.1) to materialize-0.7.1-vendor.tar.gz/crc32fast
   Vendoring criterion v0.3.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/criterion-0.3.3) to materialize-0.7.1-vendor.tar.gz/criterion
   Vendoring criterion-plot v0.4.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/criterion-plot-0.4.3) to materialize-0.7.1-vendor.tar.gz/criterion-plot
   Vendoring crossbeam-channel v0.5.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/crossbeam-channel-0.5.0) to materialize-0.7.1-vendor.tar.gz/crossbeam-channel
   Vendoring crossbeam-deque v0.8.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/crossbeam-deque-0.8.0) to materialize-0.7.1-vendor.tar.gz/crossbeam-deque
   Vendoring crossbeam-epoch v0.9.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/crossbeam-epoch-0.9.1) to materialize-0.7.1-vendor.tar.gz/crossbeam-epoch
   Vendoring crossbeam-utils v0.8.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/crossbeam-utils-0.8.3) to materialize-0.7.1-vendor.tar.gz/crossbeam-utils
   Vendoring crypto-mac v0.10.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/crypto-mac-0.10.0) to materialize-0.7.1-vendor.tar.gz/crypto-mac
   Vendoring csv v1.1.6 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/csv-1.1.6) to materialize-0.7.1-vendor.tar.gz/csv
   Vendoring csv-core v0.1.10 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/csv-core-0.1.10) to materialize-0.7.1-vendor.tar.gz/csv-core
   Vendoring datadriven v0.5.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/datadriven-0.5.0) to materialize-0.7.1-vendor.tar.gz/datadriven
   Vendoring debugid v0.7.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/debugid-0.7.2) to materialize-0.7.1-vendor.tar.gz/debugid
   Vendoring dec v0.3.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/dec-0.3.3) to materialize-0.7.1-vendor.tar.gz/dec
   Vendoring decnumber-sys v0.1.5 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/decnumber-sys-0.1.5) to materialize-0.7.1-vendor.tar.gz/decnumber-sys
   Vendoring derivative v2.2.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/derivative-2.2.0) to materialize-0.7.1-vendor.tar.gz/derivative
   Vendoring difference v2.0.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/difference-2.0.0) to materialize-0.7.1-vendor.tar.gz/difference
   Vendoring differential-dataflow v0.12.0 (https://github.com/TimelyDataflow/differential-dataflow.git#7bc5338a) (/build/source/cargo-home.66y/git/checkouts/differential-dataflow-d065d23d797aa027/7bc5338) to materialize-0.7.1-vendor.tar.gz/differential-dataflow
   Vendoring digest v0.9.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/digest-0.9.0) to materialize-0.7.1-vendor.tar.gz/digest
   Vendoring dirs-next v2.0.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/dirs-next-2.0.0) to materialize-0.7.1-vendor.tar.gz/dirs-next
   Vendoring dirs-sys-next v0.1.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/dirs-sys-next-0.1.2) to materialize-0.7.1-vendor.tar.gz/dirs-sys-next
   Vendoring doc-comment v0.3.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/doc-comment-0.3.1) to materialize-0.7.1-vendor.tar.gz/doc-comment
   Vendoring dogsdogsdogs v0.1.0 (https://github.com/TimelyDataflow/differential-dataflow.git#7bc5338a) (/build/source/cargo-home.66y/git/checkouts/differential-dataflow-d065d23d797aa027/7bc5338/dogsdogsdogs) to materialize-0.7.1-vendor.tar.gz/dogsdogsdogs
   Vendoring duct v0.13.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/duct-0.13.3) to materialize-0.7.1-vendor.tar.gz/duct
   Vendoring either v1.5.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/either-1.5.2) to materialize-0.7.1-vendor.tar.gz/either
   Vendoring encode_unicode v0.3.6 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/encode_unicode-0.3.6) to materialize-0.7.1-vendor.tar.gz/encode_unicode
   Vendoring encoding v0.2.33 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/encoding-0.2.33) to materialize-0.7.1-vendor.tar.gz/encoding
   Vendoring encoding-index-japanese v1.20141219.5 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/encoding-index-japanese-1.20141219.5) to materialize-0.7.1-vendor.tar.gz/encoding-index-japanese
   Vendoring encoding-index-korean v1.20141219.5 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/encoding-index-korean-1.20141219.5) to materialize-0.7.1-vendor.tar.gz/encoding-index-korean
   Vendoring encoding-index-simpchinese v1.20141219.5 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/encoding-index-simpchinese-1.20141219.5) to materialize-0.7.1-vendor.tar.gz/encoding-index-simpchinese
   Vendoring encoding-index-singlebyte v1.20141219.5 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/encoding-index-singlebyte-1.20141219.5) to materialize-0.7.1-vendor.tar.gz/encoding-index-singlebyte
   Vendoring encoding-index-tradchinese v1.20141219.5 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/encoding-index-tradchinese-1.20141219.5) to materialize-0.7.1-vendor.tar.gz/encoding-index-tradchinese
   Vendoring encoding_index_tests v0.1.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/encoding_index_tests-0.1.4) to materialize-0.7.1-vendor.tar.gz/encoding_index_tests
   Vendoring encoding_rs v0.8.26 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/encoding_rs-0.8.26) to materialize-0.7.1-vendor.tar.gz/encoding_rs
   Vendoring enum-iterator v0.6.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/enum-iterator-0.6.0) to materialize-0.7.1-vendor.tar.gz/enum-iterator
   Vendoring enum-iterator-derive v0.6.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/enum-iterator-derive-0.6.0) to materialize-0.7.1-vendor.tar.gz/enum-iterator-derive
   Vendoring enum-kinds v0.5.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/enum-kinds-0.5.0) to materialize-0.7.1-vendor.tar.gz/enum-kinds
   Vendoring env_logger v0.8.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/env_logger-0.8.3) to materialize-0.7.1-vendor.tar.gz/env_logger
   Vendoring failure v0.1.8 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/failure-0.1.8) to materialize-0.7.1-vendor.tar.gz/failure
   Vendoring failure_derive v0.1.7 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/failure_derive-0.1.7) to materialize-0.7.1-vendor.tar.gz/failure_derive
   Vendoring fallible-iterator v0.2.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/fallible-iterator-0.2.0) to materialize-0.7.1-vendor.tar.gz/fallible-iterator
   Vendoring fallible-streaming-iterator v0.1.9 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/fallible-streaming-iterator-0.1.9) to materialize-0.7.1-vendor.tar.gz/fallible-streaming-iterator
   Vendoring fast-float v0.2.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/fast-float-0.2.0) to materialize-0.7.1-vendor.tar.gz/fast-float
   Vendoring filetime v0.2.14 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/filetime-0.2.14) to materialize-0.7.1-vendor.tar.gz/filetime
   Vendoring flate2 v1.0.20 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/flate2-1.0.20) to materialize-0.7.1-vendor.tar.gz/flate2
   Vendoring float-cmp v0.8.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/float-cmp-0.8.0) to materialize-0.7.1-vendor.tar.gz/float-cmp
   Vendoring fnv v1.0.7 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/fnv-1.0.7) to materialize-0.7.1-vendor.tar.gz/fnv
   Vendoring foreign-types v0.3.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/foreign-types-0.3.2) to materialize-0.7.1-vendor.tar.gz/foreign-types
   Vendoring foreign-types-shared v0.1.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/foreign-types-shared-0.1.1) to materialize-0.7.1-vendor.tar.gz/foreign-types-shared
   Vendoring form_urlencoded v1.0.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/form_urlencoded-1.0.0) to materialize-0.7.1-vendor.tar.gz/form_urlencoded
   Vendoring fs_extra v1.1.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/fs_extra-1.1.0) to materialize-0.7.1-vendor.tar.gz/fs_extra
   Vendoring fstrings v0.2.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/fstrings-0.2.3) to materialize-0.7.1-vendor.tar.gz/fstrings
   Vendoring fstrings-proc-macro v0.2.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/fstrings-proc-macro-0.2.3) to materialize-0.7.1-vendor.tar.gz/fstrings-proc-macro
   Vendoring funty v1.0.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/funty-1.0.1) to materialize-0.7.1-vendor.tar.gz/funty
   Vendoring futures v0.3.13 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/futures-0.3.13) to materialize-0.7.1-vendor.tar.gz/futures
   Vendoring futures-channel v0.3.13 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/futures-channel-0.3.13) to materialize-0.7.1-vendor.tar.gz/futures-channel
   Vendoring futures-core v0.3.13 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/futures-core-0.3.13) to materialize-0.7.1-vendor.tar.gz/futures-core
   Vendoring futures-executor v0.3.13 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/futures-executor-0.3.13) to materialize-0.7.1-vendor.tar.gz/futures-executor
   Vendoring futures-io v0.3.13 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/futures-io-0.3.13) to materialize-0.7.1-vendor.tar.gz/futures-io
   Vendoring futures-macro v0.3.13 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/futures-macro-0.3.13) to materialize-0.7.1-vendor.tar.gz/futures-macro
   Vendoring futures-sink v0.3.13 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/futures-sink-0.3.13) to materialize-0.7.1-vendor.tar.gz/futures-sink
   Vendoring futures-task v0.3.13 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/futures-task-0.3.13) to materialize-0.7.1-vendor.tar.gz/futures-task
   Vendoring futures-util v0.3.13 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/futures-util-0.3.13) to materialize-0.7.1-vendor.tar.gz/futures-util
   Vendoring generic-array v0.14.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/generic-array-0.14.4) to materialize-0.7.1-vendor.tar.gz/generic-array
   Vendoring getopts v0.2.21 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/getopts-0.2.21) to materialize-0.7.1-vendor.tar.gz/getopts
   Vendoring getrandom v0.1.14 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/getrandom-0.1.14) to materialize-0.7.1-vendor.tar.gz/getrandom-0.1.14
   Vendoring getrandom v0.2.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/getrandom-0.2.1) to materialize-0.7.1-vendor.tar.gz/getrandom
   Vendoring gimli v0.23.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/gimli-0.23.0) to materialize-0.7.1-vendor.tar.gz/gimli
   Vendoring glob v0.3.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/glob-0.3.0) to materialize-0.7.1-vendor.tar.gz/glob
   Vendoring globset v0.4.6 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/globset-0.4.6) to materialize-0.7.1-vendor.tar.gz/globset
   Vendoring h2 v0.3.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/h2-0.3.0) to materialize-0.7.1-vendor.tar.gz/h2
   Vendoring half v1.6.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/half-1.6.0) to materialize-0.7.1-vendor.tar.gz/half
   Vendoring hashbrown v0.9.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/hashbrown-0.9.1) to materialize-0.7.1-vendor.tar.gz/hashbrown
   Vendoring hashlink v0.6.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/hashlink-0.6.0) to materialize-0.7.1-vendor.tar.gz/hashlink
   Vendoring heck v0.3.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/heck-0.3.1) to materialize-0.7.1-vendor.tar.gz/heck
   Vendoring hermit-abi v0.1.6 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/hermit-abi-0.1.6) to materialize-0.7.1-vendor.tar.gz/hermit-abi
   Vendoring hex v0.4.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/hex-0.4.3) to materialize-0.7.1-vendor.tar.gz/hex
   Vendoring hex-literal v0.3.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/hex-literal-0.3.1) to materialize-0.7.1-vendor.tar.gz/hex-literal
   Vendoring hmac v0.10.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/hmac-0.10.1) to materialize-0.7.1-vendor.tar.gz/hmac
   Vendoring honggfuzz v0.5.52 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/honggfuzz-0.5.52) to materialize-0.7.1-vendor.tar.gz/honggfuzz
   Vendoring http v0.2.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/http-0.2.3) to materialize-0.7.1-vendor.tar.gz/http
   Vendoring http-body v0.4.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/http-body-0.4.0) to materialize-0.7.1-vendor.tar.gz/http-body
   Vendoring httparse v1.3.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/httparse-1.3.3) to materialize-0.7.1-vendor.tar.gz/httparse
   Vendoring httpdate v0.3.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/httpdate-0.3.2) to materialize-0.7.1-vendor.tar.gz/httpdate
   Vendoring humansize v1.1.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/humansize-1.1.0) to materialize-0.7.1-vendor.tar.gz/humansize
   Vendoring humantime v2.0.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/humantime-2.0.1) to materialize-0.7.1-vendor.tar.gz/humantime
   Vendoring hyper v0.14.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/hyper-0.14.4) to materialize-0.7.1-vendor.tar.gz/hyper
   Vendoring hyper-openssl v0.9.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/hyper-openssl-0.9.1) to materialize-0.7.1-vendor.tar.gz/hyper-openssl
   Vendoring hyper-tls v0.5.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/hyper-tls-0.5.0) to materialize-0.7.1-vendor.tar.gz/hyper-tls
   Vendoring idna v0.2.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/idna-0.2.0) to materialize-0.7.1-vendor.tar.gz/idna
   Vendoring include_dir v0.6.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/include_dir-0.6.0) to materialize-0.7.1-vendor.tar.gz/include_dir
   Vendoring include_dir_impl v0.6.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/include_dir_impl-0.6.0) to materialize-0.7.1-vendor.tar.gz/include_dir_impl
   Vendoring indexmap v1.4.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/indexmap-1.4.0) to materialize-0.7.1-vendor.tar.gz/indexmap
   Vendoring indicatif v0.15.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/indicatif-0.15.0) to materialize-0.7.1-vendor.tar.gz/indicatif
   Vendoring inotify v0.9.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/inotify-0.9.2) to materialize-0.7.1-vendor.tar.gz/inotify
   Vendoring inotify-sys v0.1.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/inotify-sys-0.1.4) to materialize-0.7.1-vendor.tar.gz/inotify-sys
   Vendoring instant v0.1.6 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/instant-0.1.6) to materialize-0.7.1-vendor.tar.gz/instant
   Vendoring ipnet v2.3.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/ipnet-2.3.0) to materialize-0.7.1-vendor.tar.gz/ipnet
   Vendoring itertools v0.9.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/itertools-0.9.0) to materialize-0.7.1-vendor.tar.gz/itertools
   Vendoring itoa v0.4.6 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/itoa-0.4.6) to materialize-0.7.1-vendor.tar.gz/itoa
   Vendoring jemalloc-ctl v0.3.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/jemalloc-ctl-0.3.3) to materialize-0.7.1-vendor.tar.gz/jemalloc-ctl
   Vendoring jemalloc-sys v0.3.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/jemalloc-sys-0.3.2) to materialize-0.7.1-vendor.tar.gz/jemalloc-sys
   Vendoring jemallocator v0.3.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/jemallocator-0.3.2) to materialize-0.7.1-vendor.tar.gz/jemallocator
   Vendoring jobserver v0.1.21 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/jobserver-0.1.21) to materialize-0.7.1-vendor.tar.gz/jobserver
   Vendoring js-sys v0.3.46 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/js-sys-0.3.46) to materialize-0.7.1-vendor.tar.gz/js-sys
   Vendoring json v0.12.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/json-0.12.4) to materialize-0.7.1-vendor.tar.gz/json
   Vendoring krb5-src v0.2.4+1.18.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/krb5-src-0.2.4+1.18.2) to materialize-0.7.1-vendor.tar.gz/krb5-src
   Vendoring lazy_static v1.4.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/lazy_static-1.4.0) to materialize-0.7.1-vendor.tar.gz/lazy_static
   Vendoring lexical-core v0.7.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/lexical-core-0.7.4) to materialize-0.7.1-vendor.tar.gz/lexical-core
   Vendoring libc v0.2.89 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/libc-0.2.89) to materialize-0.7.1-vendor.tar.gz/libc
   Vendoring libm v0.2.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/libm-0.2.1) to materialize-0.7.1-vendor.tar.gz/libm
   Vendoring libsqlite3-sys v0.20.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/libsqlite3-sys-0.20.1) to materialize-0.7.1-vendor.tar.gz/libsqlite3-sys
   Vendoring libz-sys v1.1.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/libz-sys-1.1.2) to materialize-0.7.1-vendor.tar.gz/libz-sys
   Vendoring linked-hash-map v0.5.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/linked-hash-map-0.5.3) to materialize-0.7.1-vendor.tar.gz/linked-hash-map
   Vendoring linked_hash_set v0.1.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/linked_hash_set-0.1.4) to materialize-0.7.1-vendor.tar.gz/linked_hash_set
   Vendoring lock_api v0.4.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/lock_api-0.4.2) to materialize-0.7.1-vendor.tar.gz/lock_api
   Vendoring log v0.4.14 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/log-0.4.14) to materialize-0.7.1-vendor.tar.gz/log
   Vendoring matchers v0.0.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/matchers-0.0.1) to materialize-0.7.1-vendor.tar.gz/matchers
   Vendoring matches v0.1.8 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/matches-0.1.8) to materialize-0.7.1-vendor.tar.gz/matches
   Vendoring md-5 v0.9.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/md-5-0.9.1) to materialize-0.7.1-vendor.tar.gz/md-5
   Vendoring md5 v0.7.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/md5-0.7.0) to materialize-0.7.1-vendor.tar.gz/md5
   Vendoring memchr v2.3.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/memchr-2.3.3) to materialize-0.7.1-vendor.tar.gz/memchr
   Vendoring memmap v0.7.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/memmap-0.7.0) to materialize-0.7.1-vendor.tar.gz/memmap
   Vendoring memoffset v0.6.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/memoffset-0.6.1) to materialize-0.7.1-vendor.tar.gz/memoffset
   Vendoring mime v0.3.13 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/mime-0.3.13) to materialize-0.7.1-vendor.tar.gz/mime
   Vendoring miniz_oxide v0.4.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/miniz_oxide-0.4.0) to materialize-0.7.1-vendor.tar.gz/miniz_oxide
   Vendoring mio v0.7.6 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/mio-0.7.6) to materialize-0.7.1-vendor.tar.gz/mio
   Vendoring miow v0.3.6 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/miow-0.3.6) to materialize-0.7.1-vendor.tar.gz/miow
   Vendoring native-tls v0.2.7 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/native-tls-0.2.7) to materialize-0.7.1-vendor.tar.gz/native-tls
   Vendoring nix v0.19.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/nix-0.19.1) to materialize-0.7.1-vendor.tar.gz/nix
   Vendoring nom v6.0.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/nom-6.0.1) to materialize-0.7.1-vendor.tar.gz/nom
   Vendoring normalize-line-endings v0.3.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/normalize-line-endings-0.3.0) to materialize-0.7.1-vendor.tar.gz/normalize-line-endings
   Vendoring ntapi v0.3.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/ntapi-0.3.4) to materialize-0.7.1-vendor.tar.gz/ntapi
   Vendoring num v0.2.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/num-0.2.0) to materialize-0.7.1-vendor.tar.gz/num
   Vendoring num-bigint v0.2.6 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/num-bigint-0.2.6) to materialize-0.7.1-vendor.tar.gz/num-bigint
   Vendoring num-complex v0.2.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/num-complex-0.2.4) to materialize-0.7.1-vendor.tar.gz/num-complex
   Vendoring num-integer v0.1.42 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/num-integer-0.1.42) to materialize-0.7.1-vendor.tar.gz/num-integer
   Vendoring num-iter v0.1.40 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/num-iter-0.1.40) to materialize-0.7.1-vendor.tar.gz/num-iter
   Vendoring num-rational v0.2.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/num-rational-0.2.3) to materialize-0.7.1-vendor.tar.gz/num-rational
   Vendoring num-traits v0.2.14 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/num-traits-0.2.14) to materialize-0.7.1-vendor.tar.gz/num-traits
   Vendoring num_cpus v1.13.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/num_cpus-1.13.0) to materialize-0.7.1-vendor.tar.gz/num_cpus
   Vendoring num_enum v0.5.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/num_enum-0.5.1) to materialize-0.7.1-vendor.tar.gz/num_enum
   Vendoring num_enum_derive v0.5.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/num_enum_derive-0.5.1) to materialize-0.7.1-vendor.tar.gz/num_enum_derive
   Vendoring number_prefix v0.3.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/number_prefix-0.3.0) to materialize-0.7.1-vendor.tar.gz/number_prefix
   Vendoring object v0.23.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/object-0.23.0) to materialize-0.7.1-vendor.tar.gz/object
   Vendoring once_cell v1.5.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/once_cell-1.5.2) to materialize-0.7.1-vendor.tar.gz/once_cell
   Vendoring oorandom v11.1.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/oorandom-11.1.0) to materialize-0.7.1-vendor.tar.gz/oorandom
   Vendoring opaque-debug v0.3.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/opaque-debug-0.3.0) to materialize-0.7.1-vendor.tar.gz/opaque-debug
   Vendoring openssl v0.10.32 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/openssl-0.10.32) to materialize-0.7.1-vendor.tar.gz/openssl
   Vendoring openssl-probe v0.1.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/openssl-probe-0.1.2) to materialize-0.7.1-vendor.tar.gz/openssl-probe
   Vendoring openssl-src v111.14.0+1.1.1j (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/openssl-src-111.14.0+1.1.1j) to materialize-0.7.1-vendor.tar.gz/openssl-src
   Vendoring openssl-sys v0.9.61 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/openssl-sys-0.9.61) to materialize-0.7.1-vendor.tar.gz/openssl-sys
   Vendoring ordered-float v2.1.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/ordered-float-2.1.1) to materialize-0.7.1-vendor.tar.gz/ordered-float
   Vendoring os_info v3.0.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/os_info-3.0.1) to materialize-0.7.1-vendor.tar.gz/os_info
   Vendoring os_pipe v0.9.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/os_pipe-0.9.1) to materialize-0.7.1-vendor.tar.gz/os_pipe
   Vendoring parking_lot v0.11.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/parking_lot-0.11.1) to materialize-0.7.1-vendor.tar.gz/parking_lot
   Vendoring parking_lot_core v0.8.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/parking_lot_core-0.8.3) to materialize-0.7.1-vendor.tar.gz/parking_lot_core
   Vendoring parse-zoneinfo v0.3.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/parse-zoneinfo-0.3.0) to materialize-0.7.1-vendor.tar.gz/parse-zoneinfo
   Vendoring parse_duration v2.1.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/parse_duration-2.1.0) to materialize-0.7.1-vendor.tar.gz/parse_duration
   Vendoring paste v0.1.18 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/paste-0.1.18) to materialize-0.7.1-vendor.tar.gz/paste
   Vendoring paste-impl v0.1.18 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/paste-impl-0.1.18) to materialize-0.7.1-vendor.tar.gz/paste-impl
   Vendoring pdqselect v0.1.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/pdqselect-0.1.0) to materialize-0.7.1-vendor.tar.gz/pdqselect
   Vendoring percent-encoding v2.1.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/percent-encoding-2.1.0) to materialize-0.7.1-vendor.tar.gz/percent-encoding
   Vendoring pest v2.1.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/pest-2.1.3) to materialize-0.7.1-vendor.tar.gz/pest
   Vendoring phf v0.8.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/phf-0.8.0) to materialize-0.7.1-vendor.tar.gz/phf
   Vendoring phf_codegen v0.8.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/phf_codegen-0.8.0) to materialize-0.7.1-vendor.tar.gz/phf_codegen
   Vendoring phf_generator v0.8.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/phf_generator-0.8.0) to materialize-0.7.1-vendor.tar.gz/phf_generator
   Vendoring phf_shared v0.8.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/phf_shared-0.8.0) to materialize-0.7.1-vendor.tar.gz/phf_shared
   Vendoring pin-project v1.0.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/pin-project-1.0.1) to materialize-0.7.1-vendor.tar.gz/pin-project
   Vendoring pin-project-internal v1.0.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/pin-project-internal-1.0.1) to materialize-0.7.1-vendor.tar.gz/pin-project-internal
   Vendoring pin-project-lite v0.2.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/pin-project-lite-0.2.4) to materialize-0.7.1-vendor.tar.gz/pin-project-lite
   Vendoring pin-utils v0.1.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/pin-utils-0.1.0) to materialize-0.7.1-vendor.tar.gz/pin-utils
   Vendoring pkg-config v0.3.17 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/pkg-config-0.3.17) to materialize-0.7.1-vendor.tar.gz/pkg-config
   Vendoring plotters v0.2.12 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/plotters-0.2.12) to materialize-0.7.1-vendor.tar.gz/plotters
   Vendoring postgres v0.19.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/postgres-0.19.0) to materialize-0.7.1-vendor.tar.gz/postgres
   Vendoring postgres-openssl v0.5.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/postgres-openssl-0.5.0) to materialize-0.7.1-vendor.tar.gz/postgres-openssl
   Vendoring postgres-protocol v0.6.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/postgres-protocol-0.6.0) to materialize-0.7.1-vendor.tar.gz/postgres-protocol
   Vendoring postgres-types v0.2.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/postgres-types-0.2.0) to materialize-0.7.1-vendor.tar.gz/postgres-types
   Vendoring postgres_array v0.11.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/postgres_array-0.11.0) to materialize-0.7.1-vendor.tar.gz/postgres_array
   Vendoring pprof v0.4.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/pprof-0.4.2) to materialize-0.7.1-vendor.tar.gz/pprof
   Vendoring ppv-lite86 v0.2.10 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/ppv-lite86-0.2.10) to materialize-0.7.1-vendor.tar.gz/ppv-lite86
   Vendoring predicates v1.0.7 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/predicates-1.0.7) to materialize-0.7.1-vendor.tar.gz/predicates
   Vendoring predicates-core v1.0.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/predicates-core-1.0.0) to materialize-0.7.1-vendor.tar.gz/predicates-core
   Vendoring predicates-tree v1.0.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/predicates-tree-1.0.0) to materialize-0.7.1-vendor.tar.gz/predicates-tree
   Vendoring proc-macro-crate v0.1.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/proc-macro-crate-0.1.4) to materialize-0.7.1-vendor.tar.gz/proc-macro-crate
   Vendoring proc-macro-error v1.0.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/proc-macro-error-1.0.4) to materialize-0.7.1-vendor.tar.gz/proc-macro-error
   Vendoring proc-macro-error-attr v1.0.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/proc-macro-error-attr-1.0.4) to materialize-0.7.1-vendor.tar.gz/proc-macro-error-attr
   Vendoring proc-macro-hack v0.5.19 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/proc-macro-hack-0.5.19) to materialize-0.7.1-vendor.tar.gz/proc-macro-hack
   Vendoring proc-macro-nested v0.1.6 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/proc-macro-nested-0.1.6) to materialize-0.7.1-vendor.tar.gz/proc-macro-nested
   Vendoring proc-macro2 v1.0.24 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/proc-macro2-1.0.24) to materialize-0.7.1-vendor.tar.gz/proc-macro2
   Vendoring procfs v0.9.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/procfs-0.9.1) to materialize-0.7.1-vendor.tar.gz/procfs
   Vendoring prometheus v0.10.0 (https://github.com/MaterializeInc/rust-prometheus.git#8bd8207f) (/build/source/cargo-home.66y/git/checkouts/rust-prometheus-9f5e6e048c8b84a0/8bd8207) to materialize-0.7.1-vendor.tar.gz/prometheus
   Vendoring prometheus-static-metric v0.5.0-rc.1 (https://github.com/MaterializeInc/rust-prometheus.git#8bd8207f) (/build/source/cargo-home.66y/git/checkouts/rust-prometheus-9f5e6e048c8b84a0/8bd8207/static-metric) to materialize-0.7.1-vendor.tar.gz/prometheus-static-metric
   Vendoring proptest v0.10.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/proptest-0.10.1) to materialize-0.7.1-vendor.tar.gz/proptest
   Vendoring protobuf v2.17.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/protobuf-2.17.0) to materialize-0.7.1-vendor.tar.gz/protobuf
   Vendoring protobuf-codegen v2.17.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/protobuf-codegen-2.17.0) to materialize-0.7.1-vendor.tar.gz/protobuf-codegen
   Vendoring protobuf-codegen-pure v2.17.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/protobuf-codegen-pure-2.17.0) to materialize-0.7.1-vendor.tar.gz/protobuf-codegen-pure
   Vendoring psm v0.1.9 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/psm-0.1.9) to materialize-0.7.1-vendor.tar.gz/psm
   Vendoring quick-error v1.2.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/quick-error-1.2.3) to materialize-0.7.1-vendor.tar.gz/quick-error
   Vendoring quote v1.0.9 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/quote-1.0.9) to materialize-0.7.1-vendor.tar.gz/quote
   Vendoring radium v0.5.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/radium-0.5.3) to materialize-0.7.1-vendor.tar.gz/radium
   Vendoring rand v0.7.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/rand-0.7.3) to materialize-0.7.1-vendor.tar.gz/rand-0.7.3
   Vendoring rand v0.8.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/rand-0.8.3) to materialize-0.7.1-vendor.tar.gz/rand
   Vendoring rand_chacha v0.2.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/rand_chacha-0.2.2) to materialize-0.7.1-vendor.tar.gz/rand_chacha-0.2.2
   Vendoring rand_chacha v0.3.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/rand_chacha-0.3.0) to materialize-0.7.1-vendor.tar.gz/rand_chacha
   Vendoring rand_core v0.5.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/rand_core-0.5.1) to materialize-0.7.1-vendor.tar.gz/rand_core-0.5.1
   Vendoring rand_core v0.6.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/rand_core-0.6.2) to materialize-0.7.1-vendor.tar.gz/rand_core
   Vendoring rand_distr v0.4.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/rand_distr-0.4.0) to materialize-0.7.1-vendor.tar.gz/rand_distr
   Vendoring rand_hc v0.2.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/rand_hc-0.2.0) to materialize-0.7.1-vendor.tar.gz/rand_hc-0.2.0
   Vendoring rand_hc v0.3.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/rand_hc-0.3.0) to materialize-0.7.1-vendor.tar.gz/rand_hc
   Vendoring rand_pcg v0.2.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/rand_pcg-0.2.1) to materialize-0.7.1-vendor.tar.gz/rand_pcg
   Vendoring rand_xorshift v0.2.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/rand_xorshift-0.2.0) to materialize-0.7.1-vendor.tar.gz/rand_xorshift
   Vendoring rayon v1.5.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/rayon-1.5.0) to materialize-0.7.1-vendor.tar.gz/rayon
   Vendoring rayon-core v1.9.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/rayon-core-1.9.0) to materialize-0.7.1-vendor.tar.gz/rayon-core
   Vendoring rdkafka v0.26.0 (https://github.com/fede1024/rust-rdkafka.git#5d23e82a) (/build/source/cargo-home.66y/git/checkouts/rust-rdkafka-34d944143d921bb0/5d23e82) to materialize-0.7.1-vendor.tar.gz/rdkafka
   Vendoring rdkafka-sys v4.0.0+1.6.1 (https://github.com/fede1024/rust-rdkafka.git#5d23e82a) (/build/source/cargo-home.66y/git/checkouts/rust-rdkafka-34d944143d921bb0/5d23e82/rdkafka-sys) to materialize-0.7.1-vendor.tar.gz/rdkafka-sys
   Vendoring redox_syscall v0.2.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/redox_syscall-0.2.4) to materialize-0.7.1-vendor.tar.gz/redox_syscall
   Vendoring redox_users v0.4.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/redox_users-0.4.0) to materialize-0.7.1-vendor.tar.gz/redox_users
   Vendoring regex v1.4.5 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/regex-1.4.5) to materialize-0.7.1-vendor.tar.gz/regex
   Vendoring regex-automata v0.1.9 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/regex-automata-0.1.9) to materialize-0.7.1-vendor.tar.gz/regex-automata
   Vendoring regex-syntax v0.6.23 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/regex-syntax-0.6.23) to materialize-0.7.1-vendor.tar.gz/regex-syntax
   Vendoring remove_dir_all v0.5.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/remove_dir_all-0.5.1) to materialize-0.7.1-vendor.tar.gz/remove_dir_all
   Vendoring reqwest v0.11.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/reqwest-0.11.1) to materialize-0.7.1-vendor.tar.gz/reqwest
   Vendoring rlimit v0.5.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/rlimit-0.5.4) to materialize-0.7.1-vendor.tar.gz/rlimit
   Vendoring rusoto_core v0.46.0 (https://github.com/rusoto/rusoto.git#31cf1506) (/build/source/cargo-home.66y/git/checkouts/rusoto-761d9ff5e4550023/31cf150/rusoto/core) to materialize-0.7.1-vendor.tar.gz/rusoto_core
   Vendoring rusoto_credential v0.46.0 (https://github.com/rusoto/rusoto.git#31cf1506) (/build/source/cargo-home.66y/git/checkouts/rusoto-761d9ff5e4550023/31cf150/rusoto/credential) to materialize-0.7.1-vendor.tar.gz/rusoto_credential
   Vendoring rusoto_kinesis v0.46.0 (https://github.com/rusoto/rusoto.git#31cf1506) (/build/source/cargo-home.66y/git/checkouts/rusoto-761d9ff5e4550023/31cf150/rusoto/services/kinesis) to materialize-0.7.1-vendor.tar.gz/rusoto_kinesis
   Vendoring rusoto_s3 v0.46.0 (https://github.com/rusoto/rusoto.git#31cf1506) (/build/source/cargo-home.66y/git/checkouts/rusoto-761d9ff5e4550023/31cf150/rusoto/services/s3) to materialize-0.7.1-vendor.tar.gz/rusoto_s3
   Vendoring rusoto_signature v0.46.0 (https://github.com/rusoto/rusoto.git#31cf1506) (/build/source/cargo-home.66y/git/checkouts/rusoto-761d9ff5e4550023/31cf150/rusoto/signature) to materialize-0.7.1-vendor.tar.gz/rusoto_signature
   Vendoring rusoto_sqs v0.46.0 (https://github.com/rusoto/rusoto.git#31cf1506) (/build/source/cargo-home.66y/git/checkouts/rusoto-761d9ff5e4550023/31cf150/rusoto/services/sqs) to materialize-0.7.1-vendor.tar.gz/rusoto_sqs
   Vendoring rusoto_sts v0.46.0 (https://github.com/rusoto/rusoto.git#31cf1506) (/build/source/cargo-home.66y/git/checkouts/rusoto-761d9ff5e4550023/31cf150/rusoto/services/sts) to materialize-0.7.1-vendor.tar.gz/rusoto_sts
   Vendoring rusqlite v0.24.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/rusqlite-0.24.2) to materialize-0.7.1-vendor.tar.gz/rusqlite
   Vendoring rustc-demangle v0.1.16 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/rustc-demangle-0.1.16) to materialize-0.7.1-vendor.tar.gz/rustc-demangle
   Vendoring rustc_version v0.3.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/rustc_version-0.3.0) to materialize-0.7.1-vendor.tar.gz/rustc_version
   Vendoring rusty-fork v0.3.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/rusty-fork-0.3.0) to materialize-0.7.1-vendor.tar.gz/rusty-fork
   Vendoring ryu v1.0.5 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/ryu-1.0.5) to materialize-0.7.1-vendor.tar.gz/ryu
   Vendoring same-file v1.0.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/same-file-1.0.4) to materialize-0.7.1-vendor.tar.gz/same-file
   Vendoring sasl2-sys v0.1.11 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/sasl2-sys-0.1.11) to materialize-0.7.1-vendor.tar.gz/sasl2-sys
   Vendoring schannel v0.1.18 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/schannel-0.1.18) to materialize-0.7.1-vendor.tar.gz/schannel
   Vendoring scopeguard v1.1.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/scopeguard-1.1.0) to materialize-0.7.1-vendor.tar.gz/scopeguard
   Vendoring security-framework v2.0.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/security-framework-2.0.0) to materialize-0.7.1-vendor.tar.gz/security-framework
   Vendoring security-framework-sys v2.0.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/security-framework-sys-2.0.0) to materialize-0.7.1-vendor.tar.gz/security-framework-sys
   Vendoring semver v0.11.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/semver-0.11.0) to materialize-0.7.1-vendor.tar.gz/semver
   Vendoring semver-parser v0.10.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/semver-parser-0.10.1) to materialize-0.7.1-vendor.tar.gz/semver-parser
   Vendoring serde v1.0.124 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/serde-1.0.124) to materialize-0.7.1-vendor.tar.gz/serde
   Vendoring serde-protobuf v0.8.2-alpha.0 (https://github.com/MaterializeInc/serde-protobuf.git?branch=add-iter-messages#fae1e030) (/build/source/cargo-home.66y/git/checkouts/serde-protobuf-e01da57da8b70624/fae1e03) to materialize-0.7.1-vendor.tar.gz/serde-protobuf
   Vendoring serde-value v0.7.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/serde-value-0.7.0) to materialize-0.7.1-vendor.tar.gz/serde-value
   Vendoring serde_cbor v0.11.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/serde_cbor-0.11.1) to materialize-0.7.1-vendor.tar.gz/serde_cbor
   Vendoring serde_derive v1.0.124 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/serde_derive-1.0.124) to materialize-0.7.1-vendor.tar.gz/serde_derive
   Vendoring serde_json v1.0.64 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/serde_json-1.0.64) to materialize-0.7.1-vendor.tar.gz/serde_json
   Vendoring serde_regex v1.1.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/serde_regex-1.1.0) to materialize-0.7.1-vendor.tar.gz/serde_regex
   Vendoring serde_urlencoded v0.7.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/serde_urlencoded-0.7.0) to materialize-0.7.1-vendor.tar.gz/serde_urlencoded
   Vendoring sha-1 v0.9.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/sha-1-0.9.4) to materialize-0.7.1-vendor.tar.gz/sha-1
   Vendoring sha2 v0.9.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/sha2-0.9.3) to materialize-0.7.1-vendor.tar.gz/sha2
   Vendoring sharded-slab v0.1.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/sharded-slab-0.1.1) to materialize-0.7.1-vendor.tar.gz/sharded-slab
   Vendoring shared_child v0.3.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/shared_child-0.3.4) to materialize-0.7.1-vendor.tar.gz/shared_child
   Vendoring shell-words v1.0.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/shell-words-1.0.0) to materialize-0.7.1-vendor.tar.gz/shell-words
   Vendoring shlex v0.1.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/shlex-0.1.1) to materialize-0.7.1-vendor.tar.gz/shlex
   Vendoring signal-hook-registry v1.2.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/signal-hook-registry-1.2.0) to materialize-0.7.1-vendor.tar.gz/signal-hook-registry
   Vendoring siphasher v0.3.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/siphasher-0.3.3) to materialize-0.7.1-vendor.tar.gz/siphasher
   Vendoring slab v0.4.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/slab-0.4.2) to materialize-0.7.1-vendor.tar.gz/slab
   Vendoring smallvec v1.6.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/smallvec-1.6.1) to materialize-0.7.1-vendor.tar.gz/smallvec
   Vendoring snap v1.0.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/snap-1.0.4) to materialize-0.7.1-vendor.tar.gz/snap
   Vendoring socket2 v0.3.19 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/socket2-0.3.19) to materialize-0.7.1-vendor.tar.gz/socket2
   Vendoring stable_deref_trait v1.2.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/stable_deref_trait-1.2.0) to materialize-0.7.1-vendor.tar.gz/stable_deref_trait
   Vendoring stacker v0.1.13 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/stacker-0.1.13) to materialize-0.7.1-vendor.tar.gz/stacker
   Vendoring static_assertions v1.1.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/static_assertions-1.1.0) to materialize-0.7.1-vendor.tar.gz/static_assertions
   Vendoring stringprep v0.1.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/stringprep-0.1.2) to materialize-0.7.1-vendor.tar.gz/stringprep
   Vendoring strsim v0.8.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/strsim-0.8.0) to materialize-0.7.1-vendor.tar.gz/strsim
   Vendoring structopt v0.3.21 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/structopt-0.3.21) to materialize-0.7.1-vendor.tar.gz/structopt
   Vendoring structopt-derive v0.4.14 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/structopt-derive-0.4.14) to materialize-0.7.1-vendor.tar.gz/structopt-derive
   Vendoring subtle v2.2.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/subtle-2.2.3) to materialize-0.7.1-vendor.tar.gz/subtle
   Vendoring symbolic-common v8.0.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/symbolic-common-8.0.0) to materialize-0.7.1-vendor.tar.gz/symbolic-common
   Vendoring symbolic-demangle v8.0.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/symbolic-demangle-8.0.0) to materialize-0.7.1-vendor.tar.gz/symbolic-demangle
   Vendoring syn v1.0.62 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/syn-1.0.62) to materialize-0.7.1-vendor.tar.gz/syn
   Vendoring synstructure v0.12.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/synstructure-0.12.3) to materialize-0.7.1-vendor.tar.gz/synstructure
   Vendoring sysctl v0.4.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/sysctl-0.4.0) to materialize-0.7.1-vendor.tar.gz/sysctl
   Vendoring sysinfo v0.16.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/sysinfo-0.16.4) to materialize-0.7.1-vendor.tar.gz/sysinfo
   Vendoring tap v1.0.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/tap-1.0.0) to materialize-0.7.1-vendor.tar.gz/tap
   Vendoring tar v0.4.33 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/tar-0.4.33) to materialize-0.7.1-vendor.tar.gz/tar
   Vendoring tempfile v3.2.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/tempfile-3.2.0) to materialize-0.7.1-vendor.tar.gz/tempfile
   Vendoring termcolor v1.1.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/termcolor-1.1.2) to materialize-0.7.1-vendor.tar.gz/termcolor
   Vendoring terminal_size v0.1.16 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/terminal_size-0.1.16) to materialize-0.7.1-vendor.tar.gz/terminal_size
   Vendoring textwrap v0.11.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/textwrap-0.11.0) to materialize-0.7.1-vendor.tar.gz/textwrap
   Vendoring thiserror v1.0.20 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/thiserror-1.0.20) to materialize-0.7.1-vendor.tar.gz/thiserror
   Vendoring thiserror-impl v1.0.20 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/thiserror-impl-1.0.20) to materialize-0.7.1-vendor.tar.gz/thiserror-impl
   Vendoring thread_local v1.0.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/thread_local-1.0.1) to materialize-0.7.1-vendor.tar.gz/thread_local
   Vendoring timely v0.12.0 (https://github.com/TimelyDataflow/timely-dataflow#ac0a326b) (/build/source/cargo-home.66y/git/checkouts/timely-dataflow-4c0cc365061cd263/ac0a326/timely) to materialize-0.7.1-vendor.tar.gz/timely
   Vendoring timely_bytes v0.12.0 (https://github.com/TimelyDataflow/timely-dataflow#ac0a326b) (/build/source/cargo-home.66y/git/checkouts/timely-dataflow-4c0cc365061cd263/ac0a326/bytes) to materialize-0.7.1-vendor.tar.gz/timely_bytes
   Vendoring timely_communication v0.12.0 (https://github.com/TimelyDataflow/timely-dataflow#ac0a326b) (/build/source/cargo-home.66y/git/checkouts/timely-dataflow-4c0cc365061cd263/ac0a326/communication) to materialize-0.7.1-vendor.tar.gz/timely_communication
   Vendoring timely_logging v0.12.0 (https://github.com/TimelyDataflow/timely-dataflow#ac0a326b) (/build/source/cargo-home.66y/git/checkouts/timely-dataflow-4c0cc365061cd263/ac0a326/logging) to materialize-0.7.1-vendor.tar.gz/timely_logging
   Vendoring tinytemplate v1.1.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/tinytemplate-1.1.0) to materialize-0.7.1-vendor.tar.gz/tinytemplate
   Vendoring tokio v1.3.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/tokio-1.3.0) to materialize-0.7.1-vendor.tar.gz/tokio
   Vendoring tokio-macros v1.1.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/tokio-macros-1.1.0) to materialize-0.7.1-vendor.tar.gz/tokio-macros
   Vendoring tokio-native-tls v0.3.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/tokio-native-tls-0.3.0) to materialize-0.7.1-vendor.tar.gz/tokio-native-tls
   Vendoring tokio-openssl v0.6.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/tokio-openssl-0.6.1) to materialize-0.7.1-vendor.tar.gz/tokio-openssl
   Vendoring tokio-postgres v0.7.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/tokio-postgres-0.7.0) to materialize-0.7.1-vendor.tar.gz/tokio-postgres
   Vendoring tokio-stream v0.1.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/tokio-stream-0.1.4) to materialize-0.7.1-vendor.tar.gz/tokio-stream
   Vendoring tokio-util v0.6.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/tokio-util-0.6.4) to materialize-0.7.1-vendor.tar.gz/tokio-util
   Vendoring toml v0.5.8 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/toml-0.5.8) to materialize-0.7.1-vendor.tar.gz/toml
   Vendoring tower-layer v0.3.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/tower-layer-0.3.1) to materialize-0.7.1-vendor.tar.gz/tower-layer
   Vendoring tower-service v0.3.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/tower-service-0.3.0) to materialize-0.7.1-vendor.tar.gz/tower-service
   Vendoring tracing v0.1.25 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/tracing-0.1.25) to materialize-0.7.1-vendor.tar.gz/tracing
   Vendoring tracing-attributes v0.1.13 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/tracing-attributes-0.1.13) to materialize-0.7.1-vendor.tar.gz/tracing-attributes
   Vendoring tracing-core v0.1.17 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/tracing-core-0.1.17) to materialize-0.7.1-vendor.tar.gz/tracing-core
   Vendoring tracing-futures v0.2.5 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/tracing-futures-0.2.5) to materialize-0.7.1-vendor.tar.gz/tracing-futures
   Vendoring tracing-log v0.1.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/tracing-log-0.1.2) to materialize-0.7.1-vendor.tar.gz/tracing-log
   Vendoring tracing-subscriber v0.2.16 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/tracing-subscriber-0.2.16) to materialize-0.7.1-vendor.tar.gz/tracing-subscriber
   Vendoring treeline v0.1.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/treeline-0.1.0) to materialize-0.7.1-vendor.tar.gz/treeline
   Vendoring try-lock v0.2.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/try-lock-0.2.2) to materialize-0.7.1-vendor.tar.gz/try-lock
   Vendoring typenum v1.12.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/typenum-1.12.0) to materialize-0.7.1-vendor.tar.gz/typenum
   Vendoring ucd-trie v0.1.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/ucd-trie-0.1.3) to materialize-0.7.1-vendor.tar.gz/ucd-trie
   Vendoring unicase v2.6.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/unicase-2.6.0) to materialize-0.7.1-vendor.tar.gz/unicase
   Vendoring unicode-bidi v0.3.4 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/unicode-bidi-0.3.4) to materialize-0.7.1-vendor.tar.gz/unicode-bidi
   Vendoring unicode-normalization v0.1.12 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/unicode-normalization-0.1.12) to materialize-0.7.1-vendor.tar.gz/unicode-normalization
   Vendoring unicode-segmentation v1.6.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/unicode-segmentation-1.6.0) to materialize-0.7.1-vendor.tar.gz/unicode-segmentation
   Vendoring unicode-width v0.1.8 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/unicode-width-0.1.8) to materialize-0.7.1-vendor.tar.gz/unicode-width
   Vendoring unicode-xid v0.2.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/unicode-xid-0.2.0) to materialize-0.7.1-vendor.tar.gz/unicode-xid
   Vendoring url v2.2.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/url-2.2.1) to materialize-0.7.1-vendor.tar.gz/url
   Vendoring uuid v0.8.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/uuid-0.8.2) to materialize-0.7.1-vendor.tar.gz/uuid
   Vendoring vcpkg v0.2.8 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/vcpkg-0.2.8) to materialize-0.7.1-vendor.tar.gz/vcpkg
   Vendoring vec_map v0.8.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/vec_map-0.8.1) to materialize-0.7.1-vendor.tar.gz/vec_map
   Vendoring version_check v0.9.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/version_check-0.9.2) to materialize-0.7.1-vendor.tar.gz/version_check
   Vendoring wait-timeout v0.2.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/wait-timeout-0.2.0) to materialize-0.7.1-vendor.tar.gz/wait-timeout
   Vendoring walkdir v2.3.1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/walkdir-2.3.1) to materialize-0.7.1-vendor.tar.gz/walkdir
   Vendoring want v0.3.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/want-0.3.0) to materialize-0.7.1-vendor.tar.gz/want
   Vendoring wasi v0.9.0+wasi-snapshot-preview1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/wasi-0.9.0+wasi-snapshot-preview1) to materialize-0.7.1-vendor.tar.gz/wasi-0.9.0+wasi-snapshot-preview1
   Vendoring wasi v0.10.0+wasi-snapshot-preview1 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/wasi-0.10.0+wasi-snapshot-preview1) to materialize-0.7.1-vendor.tar.gz/wasi
   Vendoring wasm-bindgen v0.2.69 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/wasm-bindgen-0.2.69) to materialize-0.7.1-vendor.tar.gz/wasm-bindgen
   Vendoring wasm-bindgen-backend v0.2.69 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/wasm-bindgen-backend-0.2.69) to materialize-0.7.1-vendor.tar.gz/wasm-bindgen-backend
   Vendoring wasm-bindgen-futures v0.4.19 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/wasm-bindgen-futures-0.4.19) to materialize-0.7.1-vendor.tar.gz/wasm-bindgen-futures
   Vendoring wasm-bindgen-macro v0.2.69 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/wasm-bindgen-macro-0.2.69) to materialize-0.7.1-vendor.tar.gz/wasm-bindgen-macro
   Vendoring wasm-bindgen-macro-support v0.2.69 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/wasm-bindgen-macro-support-0.2.69) to materialize-0.7.1-vendor.tar.gz/wasm-bindgen-macro-support
   Vendoring wasm-bindgen-shared v0.2.69 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/wasm-bindgen-shared-0.2.69) to materialize-0.7.1-vendor.tar.gz/wasm-bindgen-shared
   Vendoring web-sys v0.3.36 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/web-sys-0.3.36) to materialize-0.7.1-vendor.tar.gz/web-sys
   Vendoring whoami v1.1.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/whoami-1.1.0) to materialize-0.7.1-vendor.tar.gz/whoami
   Vendoring winapi v0.3.8 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/winapi-0.3.8) to materialize-0.7.1-vendor.tar.gz/winapi
   Vendoring winapi-i686-pc-windows-gnu v0.4.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/winapi-i686-pc-windows-gnu-0.4.0) to materialize-0.7.1-vendor.tar.gz/winapi-i686-pc-windows-gnu
   Vendoring winapi-util v0.1.3 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/winapi-util-0.1.3) to materialize-0.7.1-vendor.tar.gz/winapi-util
   Vendoring winapi-x86_64-pc-windows-gnu v0.4.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/winapi-x86_64-pc-windows-gnu-0.4.0) to materialize-0.7.1-vendor.tar.gz/winapi-x86_64-pc-windows-gnu
   Vendoring winreg v0.7.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/winreg-0.7.0) to materialize-0.7.1-vendor.tar.gz/winreg
   Vendoring wyz v0.2.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/wyz-0.2.0) to materialize-0.7.1-vendor.tar.gz/wyz
   Vendoring xattr v0.2.2 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/xattr-0.2.2) to materialize-0.7.1-vendor.tar.gz/xattr
   Vendoring xml-rs v0.8.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/xml-rs-0.8.0) to materialize-0.7.1-vendor.tar.gz/xml-rs
   Vendoring zeroize v1.1.0 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/zeroize-1.1.0) to materialize-0.7.1-vendor.tar.gz/zeroize
   Vendoring zstd-sys v1.4.19+zstd.1.4.8 (/build/source/cargo-home.66y/registry/src/github.com-1ecc6299db9ec823/zstd-sys-1.4.19+zstd.1.4.8) to materialize-0.7.1-vendor.tar.gz/zstd-sys
To use vendored sources, add this to your .cargo/config.toml for this project:

installing
warning: skipping suspicious writable file '/nix/store/cfziwncdlhvazgckgzkwly4w640p4y7x-materialize-0.7.1-vendor.tar.gz'
```

---

This looks fine except for the warning which heavily hints to an issue with your build setup.

At this point it really looks like an issue with your build machine. Why is that file writable? Can you try running the build on a machine with a clean nix state?

I have run this build from a clean state on 3 different machines and it works fine. I'm going to also run the nix-review tool on this to quadruple check

---

I managed to reproduce this using `nix-review`! I'll investigate and update the PR shortly

---

So it turns out the hash of the vendor dependencies changes on this commit (rust 1.51 update) 6f73a8570ca0cb6d567e8eaebc59293f12bb7483

If I rebase my commits on top of that I get the hash that you suggest. If I rebase on top of the commit just before that I get my original hash. This is very strange as I didn't see any other packages having this issue. I updated this PR and I'll try to figure out what's going on separately

---

Some packages had a vendor update after the update to 1.51.